libs/core/popover/popover-service/popover.service.ts

```diff
@@ -127,6 +131,13 @@ export class PopoverService extends BasePopoverClass {
                 this._removeOverlay(this._modalBodyClass, this._modalTriggerClass);
             }
         });
+
+        // Add keydown listener for space key
+        this._renderer.listen('document', 'keydown', (event: KeyboardEvent) => {
```

Comment:
Listening for keydown events globally on the document level can lead some performance issues, if it is possible scope the event listener to the popover element directly. 
for example:
`
const popoverElement = this._elementRef.nativeElement;
this._renderer.listen(popoverElement, 'keydown', (event: KeyboardEvent) => {
    if (event.code === 'Space' && this.isOpen) {
        this._storeAndFocusFirstTabbable();
    }
});
`